sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -468,7 +468,9 @@
 
   /**
    * @since 4.2
+   * @deprecated no more used since 5.5
    */
+  @Deprecated
   String CORE_AUTHENTICATOR_LOCAL_USERS = "sonar.security.localUsers";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)